### DISCUSSION TITLE
Up and Down screen movement?

### DISCUSSION THREAD

**Discussion by Phantom93044** [ikemen-engine/Ikemen-GO#1816] [Feature Request] [answered]
### Describe the solution you'd like

I just personally feel like it would be cool to make it to where certain character can move up and down the screen in a kind of A.D.K. style or something. If that's like too much of an Openbor type things then that's fine. I just think that would create some interesting situations in game.

**Comment by K4thos** [comment, marked as answer]
already implemented, ikemen nightly does support Z axis
